static/js/components/WelcomeBanner.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+/* global SETTINGS: false */
+import React from 'react';
+
+import ProfileImage from './ProfileImage';
+import { getPreferredName } from '../util/util';
+import type { Profile } from '../flow/profileTypes';
+
+export default class WelcomeBanner extends React.Component {
+  props: {
+    profile:      Profile,
+    text:         string,
+    children?:    React$Element[],
+  };
+
+  render() {
+    const { profile } = this.props;
+    return (
+      <div className="welcome-banner">
+        <ProfileImage profile={profile} />
+        <div className="banner-text">
+          Hi
+          <span className="bold">
+            { ` ${getPreferredName(profile)}! ` }
+          </span>
+          Complete the sign up to enroll in the MITx
+          Micromasters for 
+          <span className="bold">
+            { " Supply Chain Management." }
```

Comment:
I think we should remove the reference to Supply Chain Management here since we don't know what to put here until they've filled out this profile after we add in the program question